datafusion/proto/src/lib.rs

```diff
@@ -422,6 +425,23 @@ mod roundtrip_tests {
             ScalarValue::TimestampSecond(Some(i64::MAX), None),
             ScalarValue::TimestampSecond(Some(0), Some("UTC".to_string())),
             ScalarValue::TimestampSecond(None, None),
+            ScalarValue::IntervalDayTime(Some(IntervalDayTimeType::make_value(0, 0))),
```

Comment:
It appears there were no round trip tests for IntervalDayTime either, so I added them as well